src/main/java/frc/robot/input/JoystickDualControllerInput.java

```diff
@@ -135,4 +135,19 @@ public boolean getSaveIntakeSetpoint() {
     public boolean getHandoff() {
         return armController.getAButton();
     }
+
+    @Override
+    public double getLeftClimbRequest() {
+        return (armController.getLeftTriggerAxis() - 0.5) * 2;
+    }
+
+    @Override
+    public double getRightClimbRequest() {
+        return (armController.getRightTriggerAxis() - 0.5) * 2;
+    }
```

Comment:
This maps the range of each trigger from -1 to 1. This means that when the driver is not pulling on the triggers, the climber will receive an input of -1. This means that the motor will be running at full reverse the whole match unless the driver "feathers" the triggers and holds them exactly in the middle of their range of travel.

This is not an acceptable user experience. The default behavior if the driver is not touching the controller should be for the climbers to stay stationary. Then we want some way to both raise and lower each climber individually.

My suggestion: have each trigger mapped to return a value between 0 and 1, such that when the trigger is not pulled it returns 0 and when the trigger is full pulled it returns 1. Then add another "reverse" button to the controller, so while the button is held, the commanded input is reversed. So if the driver pulls a trigger the climber starts going down, but if the driver also holds the "reverse" button, the climber starts going up.

Reply:
There are very few available buttons for this, but the newest commit uses pov=90 (Currently unused D-Pad button) as a reverse switch